src/LtiRegistration.php

```diff
@@ -37,7 +37,7 @@ public function getIssuer()
         return $this->issuer;
     }
 
-    public function setIssuer($issuer)
+    public function setIssuer($issuer): self
```

Comment:
Since `$this->issuer` has type `?string`, should the parameter be likewise typed as `string`?